tests/test_api_jwt.py

```diff
@@ -92,7 +91,7 @@ def test_decode_with_invalid_audience_param_throws_exception(self, jwt):
             jwt.decode(example_jwt, secret, audience=1)
 
         exception = context.value
-        assert str(exception) == 'audience must be a string or None'
+        assert str(exception) == 'audience must be a string, Iterable, or None'
```

Comment:
It's probably ok to just say `iterable` here instead of `Iterable`. I think most people have an understanding of what an iterable is with regards to Python.